src/common/components/Aisles/Aisles.js

```diff
@@ -11,14 +11,14 @@ export const Aisles = () => {
         <div className={styles.aisles__content}>
           <a href="http://www.google.com">See All Aisles</a>
           <ul className={styles.aisles__list}>
-            <li className={styles.aisles__list_item}>Fresh Fruits and Vegetables</li>
-            <li className={styles.aisles__list_item}>Deli</li>
-            <li className={styles.aisles__list_item}>Prepared Foods</li>
-            <li className={styles.aisles__list_item}>Bakery</li>
-            <li className={styles.aisles__list_item}>Fresh Meat and Seafood</li>
-            <li className={styles.aisles__list_item}>Dairy and Eggs</li>
-            <li className={styles.aisles__list_item}>Frozen</li>
-            <li className={styles.aisles__list_item}>Pantry</li>
+            <li className={[styles[`aisles__list--item`]]}>Fresh Fruits and Vegetables</li>
+            <li className={[styles[`aisles__list--item`]]}>Deli</li>
+            <li className={[styles[`aisles__list--item`]]}>Prepared Foods</li>
+            <li className={[styles[`aisles__list--item`]]}>Bakery</li>
+            <li className={[styles[`aisles__list--item`]]}>Fresh Meat and Seafood</li>
+            <li className={[styles[`aisles__list--item`]]}>Dairy and Eggs</li>
+            <li className={[styles[`aisles__list--item`]]}>Frozen</li>
+            <li className={[styles[`aisles__list--item`]]}>Pantry</li>
```

Comment:
This could benefit from some refactoring where it is a list of titles that are mapped over to produce each list item, e.g.:

```
const aisles = ['Frozen', 'Bakery']
return aisles.map(aisle => <li>{aisle}</li>
```

Reply:
WOAH.